packages/xstate-store/src/types.ts

```diff
@@ -265,3 +301,9 @@ export type ActorRefLike = {
   send: (...args: never) => void;
   getSnapshot: () => any;
 };
+
+export type Prop<T, K> = K extends keyof T ? T[K] : never;
+
+export interface StoreSchemas {
+  emitted?: Record<string, { _output: unknown }>;
+}
```

Comment:
TBH, I'm not sure why this is introduced here. It should be a separate followup PR. I also don't necessarily understand why this one only adds support for `schema.emitted`. What about `schema.events`?

Reply:
I'll move schema types to a separate PR